lint.sh

```diff
@@ -5,6 +5,8 @@ FILES=(
     "automol/error.py"
     "automol/util/_util.py"
     "automol/util/__init__.py"
+    "automol/util/heuristic.py"
+    "automol/util/matrix.py"
```

Comment:
These stay in, since I do still want the linting to be run on the files. I just don't want a separate page for them in the documentation.